<div id="viewJobSSRRoot">
 <div class="jobsearch-JobComponent icl-u-xs-mt--sm">
  <div id="welcomeToIndeedCard">
  </div>
  <div class="jobsearch-DesktopStickyContainerTrigger">
  </div>
  <div class="jobsearch-DesktopStickyContainer">
   <div>
    <div class="jobsearch-JobInfoHeader-title-container">
     <h1 class="icl-u-xs-mb--xs icl-u-xs-mt--none jobsearch-JobInfoHeader-title">
      Job opportunity: Junior data scientist / bioinformatics @ NIH/NINDS
     </h1>
    </div>
    <div class="jobsearch-CompanyInfoWithoutHeaderImage">
     <div>
      <div class="icl-u-xs-mt--xs icl-u-textColor--secondary jobsearch-JobInfoHeader-subtitle jobsearch-DesktopStickyContainer-subtitle">
       <div class="jobsearch-InlineCompanyRating icl-u-xs-mt--xs jobsearch-DesktopStickyContainer-companyrating">
        <div class="icl-u-lg-mr--sm icl-u-xs-mr--xs">
         Data Tecnica
        </div>
       </div>
       <div>
        Glen Echo, MD
       </div>
       <div>
        Remote
       </div>
      </div>
     </div>
    </div>
   </div>
   <div class="icl-u-lg-block icl-u-xs-hide">
    <div class="icl-u-xs-my--sm">
    </div>
   </div>
   <div id="indeedApplyExplanationContainer">
   </div>
   <div class="jobsearch-StickyContainerDivider">
    <div class="jobsearch-StickyContainerDivider-line">
    </div>
   </div>
  </div>
  <span>
  </span>
  <div class="mosaic-zone" id="mosaic-aboveExtractedJobDescription">
  </div>
  <div class="jobsearch-JobComponent-description icl-u-xs-mt--md">
   <div class="mosaic-zone" id="mosaic-jobHighlights">
   </div>
   <div class="mosaic-zone" id="mosaic-aboveFullJobDescription">
   </div>
   <div class="jobsearch-jobDescriptionText" id="jobDescriptionText">
    <div>
     <p>
      <b>
       Job opportunity: Junior data scientist / bioinformatics
      </b>
     </p>
     <p>
      Data Tecnica Int’l + NIH/NINDS
     </p>
     <p>
      Open as of February 1st, 2021.
     </p>
     <p>
      Come join our collaborative team at Data Tecnica International (DTI) by supporting two great labs at the National Institutes of Health’s National Institute of Neurodegenerative Diseases and Stroke (NINDS, Ward and Levine labs). This opportunity focuses on aiding with data processing, management and analysis of dynamic cellular data in the contexts of both cellular and animal models.
     </p>
     <p>
      The minimal qualifications of the candidate is a US citizen (or work permit holder) with some experience in bioinformatics, high performance computing, python programming and a basic knowledge of genomics. Ideally, a master’s degree in bioinformatics, neural computation, computational biology or biomedical data science would be best. This position is open immediately. This position is remote for the duration of at least one year, commensurate with ~1300 hours of work estimated for the calendar year.
     </p>
     <p>
      Please see below for a comprehensive outline of potential duties, responsibilities and background on the collaborative labs.
     </p>
     <p>
      If interested, please send: CV, recommendation contact information and a GitHub link or other code example to info@datatecnica.com.
     </p>
     <p>
     </p>
     <p>
      <b>
       Levine Lab Data Types
      </b>
      : All of these data will be generated by the Levine lab and provided to the contractor in a pre-processed form.
     </p>
     <ul>
      <li>
       <p>
        Mouse behavior
       </p>
       <ul>
        <li>
         <p>
          Body position/location – pose estimation video capture (3 cameras) followed by DeepLabCut or similar
         </p>
        </li>
        <li>
         <p>
          Muscle activity – EMG signal from multiple muscles
         </p>
        </li>
        <li>
         <p>
          Physical sensors – force transducers, accelerometer etc.
         </p>
        </li>
       </ul>
      </li>
      <li>
       <p>
        Cellular activity (post-hoc):
       </p>
       <ul>
        <li>
         <p>
          Automated localization analysis and counting of cFos positive cells from spinal cord images
         </p>
        </li>
        <li>
         <p>
          Above + analysis of co-localization of cell type specific markers
         </p>
        </li>
       </ul>
      </li>
      <li>
       <p>
        Cellular activity (dynamic):
       </p>
       <ul>
        <li>
         <p>
          Multi-unit array electrophysiology recordings of the neural activity in dozens to hundreds of units/cells in each animal
         </p>
        </li>
        <li>
         <p>
          Above + “opto-tagging” to genetically identify specific units within the
         </p>
        </li>
        <li>
         <p>
          broader population of recorded cells
         </p>
        </li>
       </ul>
      </li>
     </ul>
     <p>
     </p>
     <p>
      <b>
       Levine Lab Analysis Goals
      </b>
      :
     </p>
     <ul>
      <li>
       <p>
        Data analysis:
       </p>
       <ul>
        <li>
         <p>
          Integrate data from multiple sources into a multi-dimensional shared space
         </p>
        </li>
        <li>
         <p>
          Identify features/components (dimensionality reduction/PCA or similar)
         </p>
        </li>
        <li>
         <p>
          Machine learning to identify key features and their relationships to “anchor” data (ex. body position)
         </p>
        </li>
        <li>
         <p>
          Comparison of (2b) and (2c) in different biological conditions
         </p>
         <ul>
          <li>
           <p>
            Recording from different populations of neurons
           </p>
          </li>
          <li>
           <p>
            Upon silencing or activation of specific cell types
           </p>
          </li>
          <li>
           <p>
            Different behavioral/motivational contexts
           </p>
          </li>
          <li>
           <p>
            Developmental age
           </p>
          </li>
          <li>
           <p>
            Spinal cord injury
           </p>
          </li>
         </ul>
        </li>
        <li>
         <p>
          Develop a user-friendly interface for internal (our lab) use and data exploration
         </p>
        </li>
       </ul>
      </li>
      <li>
       <p>
        Data Modeling:
       </p>
       <ul>
        <li>
         <p>
          Physical forces/torques/biomechanics
         </p>
        </li>
        <li>
         <p>
          Neural decoding (query relationships and identify correlations between data from behavior and data from ephys)
         </p>
        </li>
        <li>
         <p>
          Dynamical systems modeling
         </p>
        </li>
        <li>
         <p>
          Long-term basic goal: theory/modeling of neural computations that underlie motor behavior
         </p>
        </li>
        <li>
         <p>
          Long-term translational goal: Use these models to predict effective targeted programs of spinal cord electrical stimulation as part of “neuro-rehabilitation” in patients with paralysis or neurodegeneration
         </p>
        </li>
       </ul>
      </li>
     </ul>
     <p>
     </p>
     <p>
      <b>
       Levine Lab SOW
      </b>
      :
     </p>
     <ul>
      <li>
       <p>
        Data analysis (as above), including informal consultation on data pre-processing.
       </p>
      </li>
      <li>
       <p>
        The contractor will interpret and communicate results of analysis in written and oral formats to team, and generate accompanying graphical visualizations of data
       </p>
      </li>
      <li>
       <p>
        The contractor will participate regularly in lab meetings and become an integrated team member
       </p>
      </li>
      <li>
       <p>
        The contractor will generate and maintain an annotated database of all raw and post-analysis data, and work with others within DTI to enable sharing of these datasets with the broader research community with portals in development by DTI
       </p>
      </li>
      <li>
       <p>
        The contractor will develop a user-friendly data commons to facilitate exploration and discovery for members of the lab, spinal cord and motor control researchers, and the larger neuroscience and robotics communities. This could be a part of the STRIDES cloud-based initiative.
       </p>
      </li>
     </ul>
     <p>
     </p>
     <p>
      <b>
       Levine Lab Background
      </b>
      :
     </p>
     <p>
      The mission of our lab is to understand how the central nervous system controls movement, with a focus on the cells and circuits of the spinal cord. We seek to understand (1) the identities of the cell “types” of the spinal cord, (2) how each cell type computes and contributes to specific features of motor behavior, and (3) how these cells are integrated into central nervous system-wide networks for complex movement. We have made exciting progress on our first goal over the past few years (Sathyamurthy and Johnson et al., 2018; Matson et al., 2018; Skinnider et al., 2020; and Russ and Patterson et al., 2020). Now, we are ready to launch a new set of projects to tackle our second goal and define how the details of each movement, each specific muscle contraction, are encoded within networks of genetically defined spinal cord neurons. Ultimately, we hope to use this knowledge to foster recovery in the context of neurodegenerative diseases of the motor system and following stroke or spinal injury.
     </p>
     <p>
      To do this, we have to develop new scientific approaches to link motor behavior and neural activity. We will perform mouse experiments while collecting high resolution quantitative data from multiple data streams. We will integrate and analyze this data to identify underlying features and relationships, “decode” neural activity, and test how these features change when we alter the activity of specific cell types. We will then build and test models of the neural computations that mediate movement. Finally, to fully leverage the power of these large and rich datasets, we will create a cloud-based open data commons to facilitate community-wide exploration and discovery.
     </p>
     <p>
     </p>
     <p>
      <b>
       Ward Lab Data Types
      </b>
      :
     </p>
     <ul>
      <li>
       <p>
        Proteomics: whole cell proteomics, affinity-purificaiton proteomics, “exotic” proximity labeling and subcellular fractionation proteomics
       </p>
      </li>
      <li>
       <p>
        Transcriptomics: whole cell and scRNAseq, splicing analysis, “exotic” RNAseq derivatives such as proximity labeling and ribotag
       </p>
      </li>
      <li>
       <p>
        Functional genomics: CRISPRi/a pooled screens, perturb-seq
       </p>
      </li>
      <li>
       <p>
        Microscopy: Longitudinal microscopy analysis (neuronal survival, neuritic branching), CellProfiler, “exotic” cellpaint
       </p>
      </li>
     </ul>
     <p>
     </p>
     <p>
      <b>
       Ward Lab Analysis Goals
      </b>
      :
     </p>
     <ul>
      <li>
       <p>
        Majority Effort: The contractor will primarily use existing analysis pipelines, modifying as needed for the desired applications to perform primary analysis interpretation, and visualization of proteomic, transcriptomic, genomic, functional genomic, and microscopy datasets.
       </p>
      </li>
      <li>
       <p>
        Minority Effort: Higher-order secondary analysis of above datasets
       </p>
       <ul>
        <li>
         <p>
          In the context of each-other, e.g. multi-layered analysis of related proteomic, transcriptomic, and functional genomics datasets generated by our lab
         </p>
        </li>
        <li>
         <p>
          In the context of larger curated datasets from the community (CRISPRbrain, CZI NCDN, patient genomics, curated scRNAseq datasets)
         </p>
        </li>
       </ul>
      </li>
     </ul>
     <p>
     </p>
     <p>
      <b>
       Ward Lab SOW
      </b>
     </p>
     <ul>
      <li>
       <p>
        Data analysis (as above)
       </p>
      </li>
      <li>
       <p>
        The contractor will interpret and communicate results of analysis in written and oral formats to team, and generate accompanying graphical visualizations of data
       </p>
      </li>
      <li>
       <p>
        The contractor will generate and maintain an annotated database of all raw and post-analysis data, and work with others within DTI to enable sharing of these datasets with the broader research community with portals in development by DTI
       </p>
      </li>
      <li>
       <p>
        The contractor will work with other team members within DTI, as needed, to harmonize data analysis pipelines within the group and among outside groups (e.g. CZI)
       </p>
      </li>
     </ul>
     <p>
     </p>
     <p>
      <b>
       Ward Lab
      </b>
      <b>
       Areas of synergy with ongoing efforts at DTI:
      </b>
     </p>
     <ul>
      <li>
       <p>
        Integrate/refine existing analysis pipelines that DTI has built for previous projects, harmonize analysis pipelines with related efforts at CARD/CZI when possible.
       </p>
      </li>
      <li>
       <p>
        Most of the wet-lab techniques and downstream analytic pipelines are also being developed for/used by iNDI, so there is the potential for synergy in both directions.
       </p>
      </li>
      <li>
       <p>
        Leverage and build upon existing relationships between Ward lab members, CARD, LNG, and DTI.
       </p>
      </li>
     </ul>
     <p>
     </p>
     <p>
      <b>
       <br/>
       Ward Lab background
      </b>
      :
     </p>
     <p>
      The Ward lab studies cellular and molecular mechanisms of neurodegenerative diseases. We exclusively use genetically-modified human iPSCs that harbor disease-related gene mutations or express functionalized transgenes as our cellular model system. iPSCs are then differentiated into disease-relevant cell types such as neurons and microglia. Almost every experiment in our lab begins with an unbiased ‘omics experiment, ranging from functional genomic screens to proteomics to transcriptomics. Our lab is closely affiliated with the iPSC Neurodegenerative Disease Initiative (iNDI), a large-scale NIH initiative that is in the process of generating hundreds of isogenic models of neurodegenerative diseases and related phenotypic datasets. We are also a part of the Chan Zuckerberg Institute Neurodegeneration Challenge Network (NDCN), an international network of who apply innovative, collaborative, and transparent research modalities to tackle these diseases. Increasingly, our experimental pipelines generate large multimodal datasets, examining the molecular consequences of a gene perturbation from numerous biological angles. In parallel, we study large sets of causal neurodegeneration-related genes in parallel with high-content ‘omic readouts, which we anticipate will lead to identification of converging molecular pathways. Results from these unbiased experiments are then refined into testable models, driving deep mechanistic dives into the underlying biological processes that underlie disease.
     </p>
    </div>
    <p>
    </p>
   </div>
   <div class="mosaic-zone" id="mosaic-belowFullJobDescription">
   </div>
  </div>
  <div id="mapRoot">
  </div>
  <div id="successfullySignedInModal">
  </div>
  <div id="thirdPartyApplyCreateAccountModal">
  </div>
  <div class="jobsearch-JobTab-content">
   <div class="mosaic-zone" id="mosaic-belowJobDescription">
   </div>
   <div class="jobsearch-JobMetadataFooter">
    <div class="icl-u-textColor--success">
     Data Tecnica
    </div>
    <div>
     30+ days ago
    </div>
    <div class="icl-u-lg-inline icl-us-xs-hide" id="originalJobLinkContainer">
     <a href="https://www.indeed.com/rc/clk?jk=4332f36b772aa96f&amp;from=vj&amp;pos=top&amp;mvj=0&amp;sjdu=YmZE5d5THV8u75cuc0H6Y26AwfY51UOGmh3Z9h4OvXieVH7-7ZARVsfbcFRm8uY4VAq81dAHm7GpE1MqdfeWCQ&amp;astse=62cdddf851141710&amp;assa=5511&amp;vjfrom=serp" rel="noopener" target="_blank">
      original job
     </a>
    </div>
    <div class="mosaic mosaic-provider-reportcontent" id="mosaic-provider-reportcontent">
     <div class="mosaic-reportcontent-wrapper button">
      <div>
       <button class="icl-Button icl-Button--secondary icl-Button--md mosaic-reportcontent-button desktop" type="button">
        <span class="mosaic-reportcontent-button-icon">
        </span>
        Report job
       </button>
      </div>
      <div class="mosaic-reportcontent-content">
      </div>
     </div>
    </div>
   </div>
  </div>
  <div class="mosaic-zone" id="mosaic-betweenJobDescriptionAndButtons">
  </div>
  <div class="mosaic mosaic-provider-assessments-intervention" id="mosaic-provider-assessments-intervention">
  </div>
  <div class="icl-Grid icl-Grid--gutters">
   <div class="jobsearch-ViewJobButtons-container icl-u-xs-my--md icl-Grid-col icl-u-xs-span12 icl-u-xs-textCenter icl-u-lg-textLeft" id="jobsearch-ViewJobButtons-container">
    <div class="mosaic-zone" id="mosaic-aboveViewjobButtons">
    </div>
    <div class="icl-u-lg-inlineBlock" id="viewJobButtonLinkContainer">
     <div class="" id="applyButtonLinkContainer">
      <div>
       <div class="icl-u-lg-hide">
        <a class="icl-Button icl-Button--primary icl-Button--lg icl-Button--block" href="https://www.indeed.com/rc/clk?jk=4332f36b772aa96f&amp;from=vj&amp;pos=bottom&amp;mvj=0&amp;sjdu=YmZE5d5THV8u75cuc0H6Y26AwfY51UOGmh3Z9h4OvXieVH7-7ZARVsfbcFRm8uY4VAq81dAHm7GpE1MqdfeWCQ&amp;astse=62cdddf851141710&amp;assa=5511&amp;vjfrom=serp" referrerpolicy="origin" rel="noopener" target="_blank">
         Apply Now
        </a>
       </div>
       <div class="icl-u-xs-hide icl-u-lg-block icl-u-lg-textCenter">
        <a class="icl-Button icl-Button--primary icl-Button--md icl-Button--block" href="https://www.indeed.com/rc/clk?jk=4332f36b772aa96f&amp;from=vj&amp;pos=bottom&amp;mvj=0&amp;sjdu=YmZE5d5THV8u75cuc0H6Y26AwfY51UOGmh3Z9h4OvXieVH7-7ZARVsfbcFRm8uY4VAq81dAHm7GpE1MqdfeWCQ&amp;astse=62cdddf851141710&amp;assa=5511&amp;vjfrom=serp" referrerpolicy="origin" rel="noopener" target="_blank">
         Apply On Company Site
        </a>
       </div>
      </div>
     </div>
    </div>
    <div class="icl-u-lg-inlineBlock" data-loginurl="https://www.indeed.com/account/login?dest=%2Fviewjob%3Fjk%3D4332f36b772aa96f%26from%3Dserp%26vjs%3D3" id="saveJobButtonContainer">
     <div class="">
      <div>
       <div>
        <div class="icl-u-lg-hide">
         <button class="icl-Button icl-Button--secondary icl-Button--lg icl-Button--block" type="button">
          <span class="icl-ButtonIcon">
           <svg aria-label="save-icon" class="icl-Icon icl-Icon--inheritColor icl-Icon--sm" focusable="false" role="img" viewbox="0 0 18 18">
            <g>
             <path d="M12.38,2.25A4.49,4.49,0,0,0,9,3.82,4.49,4.49,0,0,0,5.63,2.25,4.08,4.08,0,0,0,1.5,6.38c0,2.83,2.55,5.15,6.41,8.66L9,16l1.09-1C14,11.52,16.5,9.21,16.5,6.38A4.08,4.08,0,0,0,12.38,2.25ZM9.08,13.91L9,14l-0.08-.08C5.35,10.68,3,8.54,3,6.38A2.56,2.56,0,0,1,5.63,3.75,2.93,2.93,0,0,1,8.3,5.52H9.7a2.91,2.91,0,0,1,2.67-1.77A2.56,2.56,0,0,1,15,6.38C15,8.54,12.65,10.68,9.08,13.91Z">
             </path>
            </g>
           </svg>
          </span>
          <span>
           Save this job
          </span>
         </button>
        </div>
        <div class="icl-u-lg-block icl-u-xs-hide">
         <button class="icl-Button icl-Button--secondary icl-Button--md icl-Button--block" type="button">
          <span class="icl-ButtonIcon">
           <svg aria-label="save-icon" class="icl-Icon icl-Icon--inheritColor icl-Icon--sm" focusable="false" role="img" viewbox="0 0 18 18">
            <g>
             <path d="M12.38,2.25A4.49,4.49,0,0,0,9,3.82,4.49,4.49,0,0,0,5.63,2.25,4.08,4.08,0,0,0,1.5,6.38c0,2.83,2.55,5.15,6.41,8.66L9,16l1.09-1C14,11.52,16.5,9.21,16.5,6.38A4.08,4.08,0,0,0,12.38,2.25ZM9.08,13.91L9,14l-0.08-.08C5.35,10.68,3,8.54,3,6.38A2.56,2.56,0,0,1,5.63,3.75,2.93,2.93,0,0,1,8.3,5.52H9.7a2.91,2.91,0,0,1,2.67-1.77A2.56,2.56,0,0,1,15,6.38C15,8.54,12.65,10.68,9.08,13.91Z">
             </path>
            </g>
           </svg>
          </span>
          <span>
           Save this job
          </span>
         </button>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div class="icl-u-lg-hide" id="shareJobButton">
    </div>
    <div class="mosaic-zone" id="mosaic-belowViewjobButtons">
    </div>
    <div class="mosaic mosaic-provider-jobsearch-feedback" id="mosaic-provider-jobsearch-feedback">
    </div>
   </div>
  </div>
 </div>
 <div class="icl-Grid icl-Grid--gutters" id="relatedLinks">
  <div class="icl-Grid icl-Grid--gutters jobsearch-RelatedLinks">
   <div class="icl-Grid-col icl-u-xs-span12">
    <div class="jobsearch-RelatedLinks-linkWrapper">
     <a class="jobsearch-RelatedLinks-link" href="/jobs?q=Junior+Data+Scientist&amp;l=Glen+Echo%2C+MD">
      Junior Data Scientist jobs in Glen Echo, MD
     </a>
    </div>
    <div class="jobsearch-RelatedLinks-linkWrapper">
     <a class="jobsearch-RelatedLinks-link" href="/jobs?q=Data+Tecnica&amp;l=Glen+Echo%2C+MD">
      Jobs at Data Tecnica in Glen Echo, MD
     </a>
    </div>
    <div class="jobsearch-RelatedLinks-linkWrapper">
     <a class="jobsearch-RelatedLinks-link" href="/salary?q1=Junior+Data+Scientist&amp;l1=Glen+Echo%2C+MD">
      Junior Data Scientist salaries in Glen Echo, MD
     </a>
    </div>
   </div>
  </div>
 </div>
</div>